utils/yoke/README.md

```diff
@@ -0,0 +1,12 @@
+# yoke [![crates.io](http://meritbadge.herokuapp.com/yoke)](https://crates.io/crates/yoke)
+
+This crate provides [`Yoke`], a data structure that allows one
```

Comment:
[`Yoke`] should be provided a URL here, though it'll be tricky to make that work with the readme sync. Perhaps we can just have it not be a URL in both places?

Reply:
I think this is a pre-existing problem. Links in other auto-generated READMEs in this repo are also broken.

This file is in this PR because it was auto-generated by the Makefile command; it probably should have already been there.

Reply:
Seems fair